File: backend/app/dbconfig.py
# from pymongo import MongoClient
# uri = "mongodb://localhost:27017/"
# client = MongoClient(uri)["tasktracker"]
# db_name = "tasktracker" 
# dbclient = client[db_name]


from motor.motor_asyncio import AsyncIOMotorClient

mongourl = "mongodb+srv://sbcreations:sbcreations@cluster0.ao6t0sl.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0"

mongodb = AsyncIOMotorClient(mongourl)
# mongodb = MongoClient(mongourl)

client = mongodb['tasktracker']

tasksCollection = client["tasks"]
userCollection = client['users']




# from motor.motor_asyncio import AsyncIOMotorClient

# mongourl = "mongodb://localhost:27017"
# mongodb = AsyncIOMotorClient(mongourl)
# client = mongodb['tasktracker']
# tasksCollection = client["tasks"]
# userCollection = client['users']

File: backendNodev2/db/dbConfig.js
const mongoose = require('mongoose')
require('dotenv').config()

mongoUrl = process.env.MONGO_URL || "mongodb+srv://sbcreations:sbcreations@cluster0.ao6t0sl.mongodb.net/tasktracker?"
const conDB = async () =>{       
    await mongoose.connect(mongoUrl)
    .then(()=>{
        console.log("MONGODB Connected")
    })
    .catch((err)=>{
        console.log(`MONGODB not Connected... Error: ${err}`)
    })
}

module.exports = conDB;